staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go

```diff
@@ -210,14 +245,19 @@ func printSingleResourceUsage(out io.Writer, resourceType v1.ResourceName, quant
 	switch resourceType {
 	case v1.ResourceCPU:
 		fmt.Fprintf(out, "%vm", quantity.MilliValue())
-	case v1.ResourceMemory:
+	case v1.ResourceMemory, ResourceSwap:
 		fmt.Fprintf(out, "%vMi", quantity.Value()/(1024*1024))
 	default:
 		fmt.Fprintf(out, "%v", quantity.Value())
 	}
 }
 
-func printPodResourcesSum(out io.Writer, total v1.ResourceList, columnWidth int) {
+func printSingleMissingResource(out io.Writer) {
+	const unavailableStr = "unavailable"
```

Comment:
```suggestion
	const unavailableStr = "<unkown>"
```
is what we're mostly using, in `kubectl decribe` output, for example. So let's be consistent with that.